### ISSUE TITLE
SQLite rewrite

### ISSUE THREAD

**Issue by tonibm19** [mc-server/Essentials#25]
Rewrite jails, homes and warps to use SQLite.
I don't know SQLite, so I won't do it soon, maybe some day if I learn.
Any volunteer?

**Comment by mathias-github** [comment]
Just curious, what exactly is the advantage of SQLite?

**Comment by tonibm19** [comment]
Faster access to data.
Right now essentials is using ini files, which, specially in the case of jails, can be slow.

**Comment by NiLSPACE** [comment]
How is the jail done then? Does it check the INI file when the player moves or when the player joins? If it checks it when the player moves, then I suggest creating a cPlayerState object just like WorldEdit. In there you can save if the player is in jail or not.

**Comment by tonibm19** [comment]
I'm not quite sure, it's been a long time since I coded and tested it for the last time.
I will have a look.
Anyway, SQL would be better.

**Comment by NiLSPACE** [comment]
Yes, SQL would be better, but it's still better to save if someone is jailed in memory.
